core/build.sbt

```diff
@@ -0,0 +1,7 @@
+val sbtTypelevelVersion = "0.4.14" // https://github.com/typelevel/sbt-typelevel/releases
+
+addSbtPlugin("org.typelevel" % "sbt-typelevel-ci-release" % "0.4.14")
+addSbtPlugin("org.typelevel" % "sbt-typelevel-settings" % "0.4.14")
+addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "1.0.0") // https://github.com/scalastyle/scalastyle-sbt-plugin/tags
```

Comment:
Is scalastyle still a real maintained thing? I'd be much happier to make it go away and replace it with sbt-typelevel-scalafix. See also:
https://github.com/typelevel/typelevel-scalafix/

Reply:
Yeah, tbh I think we can just remove scalastyle. I guess part of the adoption of this plugin will be using scalafix instead of scalastyle 😄

Reply:
Removed.